packages/sku/src/services/vite/helpers/createConfig.ts

```diff
@@ -107,19 +109,20 @@ export const createViteConfig = ({
       esbuildOptions: {
         plugins: [fixViteVanillaExtractDepScanPlugin()],
       },
+      exclude: skuContext.skuConfig.skipPackageCompatibilityCompilation,
     },
     ssr: {
       external: [
-        ...builtinModules,
         '@vanilla-extract/css/adapter',
         'serialize-javascript',
         'used-styles',
         '@sku-lib/vite',
-        ...(configType === 'ssg' || configType === 'ssr'
-          ? ['sku/vite/loadable']
-          : []),
       ],
-      noExternal: ['@vanilla-extract/css', 'braid-design-system'],
+      noExternal: [
+        '@vanilla-extract/css',
```

Comment:
This a already marked as external by the Vanilla Extract plugin, so it should just be removed IMO.